resources/test-data.md

```diff
@@ -7,6 +7,17 @@ description: |
 menu_order: 1000
 ---
 
+<script language="javascript">
+const elm = document.querySelector(".doc-container");
```

Comment:
Should we add the event to the entire `.doc-container` or just the `.table` elements?

Reply:
I figured (well, Øyvind and myself) figured that maybe it was a good idea so that it will work everywhere you copy something, not just in said table?

Reply:
It doesn't work with .table   ..?